hasher-matcher-actioner/requirements.txt

```diff
@@ -1 +1 @@
-.
\ No newline at end of file
+.
```

Comment:
What change happened here?

Reply:
Probably a line ending thing. Can be ignored, I suppose.